ci/install_data_services.py

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import sys
+from concurrent.futures.thread import ThreadPoolExecutor
+
+import sdk_install
+import logging
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+def install_package(package_name, service_name):
+    log.info("Starting %s installation.", package_name)
+    sdk_install.install(package_name, service_name, 0, package_version=sdk_install.PackageVersion.LATEST_UNIVERSE)
+
+
+def main(argv) -> int:
+    executor = ThreadPoolExecutor(max_workers=10)
+    for data_service in argv[1:]:
+        executor.submit(install_package, data_service, data_service)
+
+    executor.shutdown(wait=True)
```

Ouch, I only now noticed we're effectively ignoring exceptions here too :-/

---

I don't suppress it. What would you propose here?

---

What I meant is that if you submit a task and never check its return code like here, then you are effectively ignoring its fate.
There's more than one way to skin a cat, the most obvious one to me is [using `as_completed` and the executor as context manager](https://docs.python.org/3/library/concurrent.futures.html#threadpoolexecutor-example).